src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -7556,10 +7556,14 @@ public sealed partial class AmbiguousMatchException : System.SystemException
     public abstract partial class Assembly : System.Reflection.ICustomAttributeProvider, System.Runtime.Serialization.ISerializable
     {
         protected Assembly() { }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
Asked question about this in https://github.com/dotnet/runtime/issues/31127#issuecomment-660108692

Reply:
Removed `EditorBrowsableAttribute` per discussion